Source/Core/CircleGeometry.js

```diff
@@ -174,6 +176,17 @@ define([
             vertexFormat : VertexFormat.POSITION_ONLY
         });
     };
+    
+    defineProperties(CircleGeometry.prototype, {
+        /**
+         * @private
+         */
+        rectangle : {
```

Comment:
So I started doing this and there was some name collisions. Throughout Cesium `boundingRectangle` is used to mean a rectanglular volume in Cartesian space, while `Rectangle` is a lat/lon rectangle which is what this property is. I think making it `boundingRectangle` would be more confusing, don't you?